app/src/main/res/layout/item_list.xml

```diff
@@ -20,7 +22,15 @@
         app:srcCompat="@drawable/ic_drag_vertical_white_24dp"
         tools:ignore="ContentDescription" />
 
+    <View
```

Comment:
You should put a comment here for future reference so people know what this is for.